clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -19,10 +19,6 @@ const PRIVACY_NOTICE_KEY_1 = "advertising";
 const PRIVACY_NOTICE_KEY_2 = "essential";
 const PRIVACY_NOTICE_KEY_3 = "analytics_opt_out";
 
-const PRIVACY_NOTICE_KEY_4 = "data_sales";
```

Comment:
I _think_ I reverted all these tests to expect the behaviour prior to #4527; you can see those changes in that PR